guides/common/modules/con_configuring-network-interfaces.adoc

```diff
@@ -1,31 +1,25 @@
-[id="Adding_Network_Interfaces_{context}"]
-= Adding network interfaces
+[id="configuring_network_interfaces"]
```

Comment:
```suggestion
[id="configuring-network-interfaces"]
```

This makes `grep` even more fun because you'll find the file, anchor, and all links if you drop the "con_" prefix.